src/platform/site-wide/index.js

```diff
@@ -54,21 +54,21 @@ export default function startSitewideComponents(commonStore) {
    * Once https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/3568 is complete
    * this block can be removed
    */
-  if (
-    document.readState === 'complete' ||
-    document.readState === 'loaded' ||
-    document.readState === 'interactive'
```

Comment:
Looks like the old code was looking for `document.readState` instead of `document.readyState`, was there any testing done to see if just fixing this typo effected the same change? Just curious.

Reply:
I did fix the typo but found some unexpected functionality was introduced. Specifically, in this if statement, `readyState` will always evaluate to `interactive` which led to a weird race condition where handlers weren't actually attaching properly. Because of this, I just went with a refactor.